src/Header/Header.js

```diff
@@ -1,12 +1,19 @@
 import React from 'react'
-import './Header.css'
+import '../scss/_Header.scss'
 
-const Header = () => {
+const Header = (props) => {
```

Comment:
Maybe we could destructure the props here to more easily see what 'props' is as it's being passed into header?

Reply:
All about it!